doc/classes/CheckButton.xml

```diff
@@ -58,16 +61,28 @@
 			The [StyleBox] to display as a background.
 		</theme_item>
 		<theme_item name="off" type="Texture2D">
-			The icon to display when the [CheckButton] is unchecked.
+			The icon to display when the [CheckButton] is unchecked (for left-to-right layouts).
 		</theme_item>
 		<theme_item name="off_disabled" type="Texture2D">
-			The icon to display when the [CheckButton] is unchecked and disabled.
+			The icon to display when the [CheckButton] is unchecked and disabled (for left-to-right layouts).
+		</theme_item>
+		<theme_item name="off_disabled_mirrored" type="Texture2D">
```

Comment:
Isn't this too many items? I kind of believe that simply mirroring the icon automatically would be enough in 99% of cases. I'd rather avoid defining a huge set of properties for all theme texture that could be flipped.

However, I believe we should maybe have a more generic approach here, likely allowing for a whole "mirrored" theme system. It would kind of inherit the base theme and replace only the items you want mirrored. That would avoid defining special properties for mirrored situations.

Reply:
Only few theme items need mirroring (option button and check button), and check button have text on it, simply mirroring texture won't work. Neither all icons need to be mirrored, only arrows and other things for forward/backward indication, rest is ok as is.